docs/processing_information.md

```diff
@@ -0,0 +1,60 @@
+# Processing information
+
+## Single-cell and single-nuclei RNA-seq
+
+### Alignment and quantification using alevin-fry
+
+We used [`salmon alevin`](https://salmon.readthedocs.io/en/latest/alevin.html) and [`alevin-fry`](https://alevin-fry.readthedocs.io/en/latest/) to generate gene by cell counts matrices for all single-cell and single-nuclei samples.
+In brief, we utilized [selective alignment](#selective-alignment) to the [`splici` index](#the-splici-index) for all single-cell and single-nuclei samples. 
+
+#### Reference transcriptome index
+
+For all samples, we aligned FASTQ files to a reference transcriptome index referred to as the `splici` index.
+The [`splici` index](https://combine-lab.github.io/alevin-fry-tutorials/2021/improving-txome-specificity/) is built using transcripts from both spliced cDNA and intronic regions.
+Inclusion of intronic regions in the index used for alignment allowed us to capture both reads from mature, spliced cDNA and nascent, unspliced cDNA. 
+Alignment of RNA-sequencing data to an index containing intronic regions has been shown to reduce spuriously detected genes ([He _et al._ 2021](https://www.biorxiv.org/content/10.1101/2021.06.29.450377v1), [Kaminow _et al._ 2021](https://www.biorxiv.org/content/10.1101/2021.05.05.442755v1.full#sec-5))
+In our hands, we have found that use of the `splici` index led to a more comparable distribution of unique genes found per cell to Cell Ranger than use of an index obtained from spliced cDNA transcripts only. 
+
+#### Selective alignment
+
+We aligned reads to the transcriptome index using `salmon` with the defualt "selective alignment" strategy. 
+Briefly, selective alignment uses a mapping score validated approach to identify maximal exact matches between reads and the provided index. 
+For all samples, we used selective alignment to the `splici` index. 
+
+A more detailed description of the alignment strategy invoked by `salmon` in conjuction with `alevin-fry` can be found [here](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02151-8). 
```

Comment:
I would probably use an in text citation over `[here](<link to paper>)`.  I'll just get this out of the way and tell you that we're going to use the Genetics style citations, but I think you knew that based on the ones further up! DOI link for this one, too, please.